README.md

```diff
@@ -161,6 +161,18 @@ class User
 end
 ```
 
+To improve performance, you can save many entities at once. (Batch operation)
```

Comment:
At this position in the README we have not yet mentioned the concept of parent/ancestor. Let's move this new section down and after Datastore Consistency.